walk/walk.go

```diff
@@ -297,10 +295,7 @@ func loadBuildFile(c *config.Config, pkg, dir string, ents []fs.DirEntry) (*rule
 	return rule.LoadFile(path, pkg)
 }
 
-func configure(cexts []config.Configurer, knownDirectives map[string]bool, c *config.Config, rel string, f *rule.File) *config.Config {
-	if rel != "" {
-		c = c.Clone()
```

Comment:
Could you add a comment to this function to make it clear that `c` may be modified and should be cloned by the caller?

Reply:
Done